src/actions/actionActions.ts

```diff
@@ -110,10 +110,10 @@ export const deleteActionThunkAsync = (appId: string, actionId: string, removeFr
 
             // Fetch train dialogs if any train dialogs were impacted
             if (deleteEditResponse.trainDialogIds && deleteEditResponse.trainDialogIds.length > 0) {
-                dispatch(fetchAllTrainDialogsThunkAsync(appId));
+                await dispatch(fetchAllTrainDialogsThunkAsync(appId));
```

Comment:
Also needs to not await

Reply:
Removed await from all dispatches here, and other files as well.